File: pinyougou_web_portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){
	this.findByCategoryId = function(categoryId){
			return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
	}
	this.add=function (entity) {
		return $http.post("content/add.do",entity);

    }
    this.update=function (entity) {
		return $http.post("content/update.do",entity);

    }
    this.dele=function (ids) {
		return $http.get("content/dele.do?ids="+ids);

    }
    this.findOne=function (id) {
		return $http.get("content/findOne.do?id="+id);
    }
});

File: pinyougou_interface/src/main/java/com/pinyougou/sellergoods/service/ContentService.java
package com.pinyougou.sellergoods.service;

import cn.itcast.core.pojo.ad.Content;
import cn.itcast.core.pojo.ad.ContentCategory;
import entity.PageResult;


import java.util.List;

public interface ContentService {

    List<Content> findByCategoryId(long categoryId);


    PageResult search(Integer page, Integer rows, ContentCategory contentCategory);

    void add(Content content);

    void update(Content content);

    void dele(long[] ids);

    Content findOne(long id);
}
